internal/kokoro/synth/synth.sh

```diff
@@ -7,6 +7,13 @@
 set -e -x
 
 cd $(dirname $0)/../../..
+
+# Install Go 1.15
+curl -o /tmp/go.tgz https://dl.google.com/go/go1.15.1.linux-amd64.tar.gz && \
+    tar -C /usr/local -xzf /tmp/go.tgz && \
```

Comment:
Is Go already installed at `/usr/local`? If so, we need to delete it first.

Reply:
Done.